src/airport_gui.cpp

```diff
@@ -427,7 +428,7 @@ class BuildAirportWindow : public PickerWindowBase {
 			int rad = _settings_game.station.modified_catchment ? as->catchment : (uint)CA_UNMODIFIED;
 
 			/* only show the station (airport) noise, if the noise option is activated */
-			if (_settings_game.economy.station_noise_level) {
+			if (_settings_game.difficulty.town_council_tolerance != TOWN_COUNCIL_PERMISSIVE && _settings_game.economy.station_noise_level) {
```

Comment:
It's probably worth adding a helper function for the repeated use of this condition, something like `bool TownControlsNoiseLevel()`

Reply:
I have done this, just not pushed yet :-)

Reply:
If you push it, I'll review it. 😉

Reply:
(still willing to review this 😉 )